src/Hosting/Hosting/test/HostBuilderTests.cs

```diff
@@ -118,10 +118,8 @@ public void CanConfigureAppConfigurationAndRetrieveFromDI()
         [Fact]
         public void CanConfigureAppConfigurationFromFile()
         {
-            // Needs to look in the project directory like VS would
```

Comment:
The intent of this test has been changed. It was emulating VS behavior where these config files are not published, but now you've changed everything to only test published. Is there any way to continue to test the VS scenario?

Reply:
Can you clarify what the VS scenario is? If that scenario is "Visual Studio sets "Directory.GetCurrentDirectory()" to the project directory", then this test doesn't actually assert that.

Reply:
Yes, VS sets the current director to the project dir *and* does not publish config files by default.

Reply:
If that's what you want to test, this test needs to be completely rewritten. What it's doing is a poor proxy for that scenario. If VS decided to change the current directory, this test would still pass. If the SDK started publishing this config files, the test would still pass. Nothing in this test code is asserting the _absence_ of the file in publish output, and nothing is asserting the cwd behavior of Visual Studio.

Reply:
I would prefer to leave this change in. This change makes the test compatible with Helix.

Reply:
I won't block it, but we need to keep in mind where our tests diverge from the normal user experience.